docs/source/chapters/about.rst

```diff
@@ -14,9 +14,8 @@ About The Workshop
     :align: center
     :target: http://www.osgeo.org/
 
-This workshop uses several FOSS4G tools. Most of FOSS4G software is
-related to other open source projects and it would go too far to list them all
-here.
+This workshop uses several FOSS4G (Free and Open Source Software for Geospatial) tools. Most of the FOSS4G tools are
+related to other open-source software (OSS) projects and it would not be feasible to list all of them.
```

Comment:
open-source
or
open source

Reply:
For this, I tried to follow this: https://en.wikipedia.org/wiki/Open-source_software

Reply:
its 2 words, even OSGeo Open Source for Geospacial uses 2 words

Reply:
Sure. Will update it accordingly.